superset-frontend/src/components/AlteredSliceTag.jsx

```diff
@@ -103,10 +138,7 @@ export default class AlteredSliceTag extends React.Component {
     if (value === null) {
       return 'null';
     }
-    if (
-      this.state.controlsMap[key] &&
-      this.state.controlsMap[key].type === 'AdhocFilterControl'
-    ) {
+    if (controlsMap[key] && controlsMap[key].type === 'AdhocFilterControl') {
```

Comment:
you could use optional chaining here if you wanted to clean this up more.

Reply:
👍